src/go/cluster-to-redpanda-migration/Dockerfile

```diff
@@ -1,7 +0,0 @@
-FROM public.ecr.aws/docker/library/golang:1.20.5
```

Comment:
oh, we don't ship container for the migration?

Reply:
no, goreleaser is what we are using now.